crates/js-component-bindgen/src/ts_bindgen.rs

```diff
@@ -472,6 +476,7 @@ impl TsBindgen {
         resolve: &Resolve,
         id: InterfaceId,
         files: &mut Files,
+        guest: bool,
```

Comment:
Can `guest` be a property of the bindgen struct itself?

Reply:
Done. I initially went to do this but decided against it because of imposter syndrome 🥲. Glad to see I wasn't too far off base!